Gemfile

```diff
@@ -57,6 +57,7 @@ gem 'sitemap', git: 'https://github.com/ualbertalib/rails-sitemap.git'
 gem 'rest-client'
 
 gem 'noid', '~> 0.8'
+gem 'nokogiri', '~> 1.7.1'
```

Comment:
Just a quick comment about this is for locking nokogiri or something for security reasons? As we ourselves are not using nokogiri in this project that I know of?